Dockerfile

```diff
@@ -2,29 +2,25 @@
 FROM jupyterhub/k8s-hub:0.9.0-n033.h8211ad2
 
 USER root
-# install d4science auth plugin for using the service in D4Science
-RUN pip3 install git+https://github.com/enolfc/d4scienceauth.git
 
-# install oauthenticator with EGI Check-in, with refresh
-RUN pip3 install git+https://github.com/enolfc/egiauthenticator@refresh
+COPY . /egi-notebooks-hub/
 
-# install the EGI Spawner
-RUN pip3 install git+https://github.com/enolfc/egispawner
+# install the hub extensions
+RUN pip3 install /egi-notebooks-hub/
 
-RUN pip3 install git+https://github.com/enolfc/egi-hub-addons
+# install d4science auth plugin for using the service in D4Science
+RUN pip3 install git+https://github.com/enolfc/d4scienceauth.git
 
```

Comment:
```suggestion
RUN pip3 install git+https://github.com/enolfc/egi-hub-addons
```

Reply:
I put it back for now..

Reply:
I have added the code in a push below, needs to be tested, but should mostly work